src/Process/Coverage/CoverageRequirementChecker.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright © 2017-2018 Maks Rafalko
+ *
+ * License: https://opensource.org/licenses/BSD-3-Clause New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Infection\Process\Coverage;
+
+use Composer\XdebugHandler\XdebugHandler;
+
+/**
+ * @internal
+ */
+final class CoverageRequirementChecker
+{
+    /**
+     * @var bool
+     */
+    private $skipCoverage;
+
+    /**
+     * @var string
+     */
+    private $initialTestPhpOptions;
+
+    public function __construct(bool $skipCoverage, string $initialTestPhpOptions)
+    {
+        $this->skipCoverage = $skipCoverage;
+        $this->initialTestPhpOptions = $initialTestPhpOptions;
+    }
+
+    public function hasDebuggerOrCoverageOption(): bool
+    {
+        return $this->skipCoverage
+            || \PHP_SAPI === 'phpdbg'
```

Comment:
This and two places above also.

(Not sure if this specific line would affect MSI.)